### PR TITLE
Unique

### PR DESCRIPTION
**Pull request by clemcx**
creation of unique Id with the polyfill-uuid bundlle

### CODE REVIEW

File: assets/scss/app.css [hunk 1]
```diff
@@ -1,50 +1,17 @@
-/*
-Error: File to import not found or unreadable: ~bootstrap.
-        on line 2 of app.scss
+/* Error: Can't find stylesheet to import.
```
Comment (benj-san) [inline assets/scss/app.css]: mmh I think this comment can be removed?

File: src/Entity/Board.php [hunk 2]
```diff
@@ -38,16 +45,22 @@ class Board
      */
     private $advisors;
```
Comment (benj-san) [inline src/Entity/Board.php:46]: Can you name this attribute as a singular noun, since you'll only get one advisor when searching

**Review by benj-san — COMMENTED** [review]
All good besides those few comments, fix it then I shall approve it

**Review by RMayeux — APPROVED** [review]
It's good for me

File: src/Entity/Board.php (hunk 2 above)
Comment (clemcx) [reply src/Entity/Board.php:46]: it's a many to many relation, you'll get an array of advisors. I think the plural is relevant

**Review by benj-san — APPROVED** [review]

**Pull request merged**